### ISSUE TITLE
New Uservoice Issue: HRTB.US System down?

### ISSUE THREAD

**Issue by BretFisher** [Code4HR/hrt-bus-finder#198]
I have not been able to get any results, just an opening "refresh" page for two days now.

I hope the system will be back up... I rely on this app almsot daily... It is VERY helpful!

http://hrtbus.uservoice.com/admin/tickets/40

**Comment by bschoenfeld** [comment]
### Step 1
- Check https://hrtb.us/#/routes/
- No routes in the dropdown, which is a problem.
- There are bus icons on the map with recent checkin times, which indicates the HRT data feed is running and our server is processing it.
- Indicative of a problem with the GTFS schedule data

### Step 2
- Log on to server
- `cd hrt-bus-api-logs`
- `cat` most recent logs from both ftp and gtfs process
- No errors and none of the stats are zero, so on the surface, it seems everything is running normally.

### Step 3
- Looking through front end code, it seems route dropdown is populated by calling `/api/routes/active/`. Find implementation in backend code and investigate database records being called by using mongo admin on server. Find that code is calling find on a collection that isn't in the database. This collection is supposed to be created each day by the GTFS script.
- Run the GTFS script manually for today `python process-gtfs.py 0`. Got an error that wasn't saved in the logs.

```
Traceback (most recent call last):
  File "process-gtfs.py", line 89, in <module>
    'stopId': row['stop_id'],
KeyError: 'stop_id'
```
- Will investigate soon.

### TODO
- Document how healthy script logs are supposed to look
- Document how to run script in README
- Make sure script errors are logged

**Comment by bschoenfeld** [comment]
### Resolution
GTFS files now contains UTF-8 BOM in the beginning, probably by mistake. Update the script to remove these characters http://stackoverflow.com/a/18664752/438281

Fixed by Code4HR/hrt-bus-api#83
